src/Makefile

```diff
@@ -2,4 +2,8 @@ INCLUDE_DIRS = ../inc
 CFLAGS += -O2
 
 c_kzg_4844.o: c_kzg_4844.c Makefile
-	clang -Wall -I$(INCLUDE_DIRS) $(CFLAGS) -c $<
+	clang -Wall -I$(INCLUDE_DIRS) $(CFLAGS) -mmacosx-version-min=10.15 -c $<
```

Comment:
that mmacosx flag can't be checked in -- we have to detect platform or something